third_party/docfx-doclet-143274/src/main/java/com/microsoft/lookup/BaseLookup.java

```diff
@@ -32,6 +33,7 @@
 
 public abstract class BaseLookup<T extends Element> {
 
+    private static final int INITIAL_CAPACITY = 500000;

```

Comment:
Set large initial map capacity to *try* and reduce the number of copy/ increases needed. This is for ClassLookup, ClassItemsLookup, and PackageLookup. Maybe I can allow configuration of this capacity for the different lookups. ClassLookup (mid size), ClassItemsLookup (large), PackageLookup (small)